services/essentials.yaml

```diff
@@ -68,13 +68,19 @@ spec:
       version: '1.3'
     - groupId: org.jenkins-ci.plugins.icon-shim
       artifactId: icon-shim
-      version: '2.0.3'
+      version: 2.0.3
     - groupId: org.jenkins-ci.plugins
       artifactId: jsch
-      version: '0.1.54.2'
+      version: 0.1.54.2
     - groupId: org.jenkins-ci.plugins
       artifactId: script-security
-      version: '1.47-rc766.cbcb5e5e6008'
+      version: '1.46'
```

Comment:
Reverting script-security to 1.46 non incrementals to avoid

```
make : on entre dans le répertoire « /home/tiste/dev/github/jenkins-infra/evergreen/services »
node ./prepare-essentials save
info: Resolving dependencies...
(node:1) UnhandledPromiseRejectionWarning: Error: Invalid argument not valid semver ('1.47-rc766.cbcb5e5e6008' received)
    at validate (/home/tiste/dev/github/jenkins-infra/evergreen/services/node_modules/compare-versions/index.js:36:13)
    at Array.forEach (<anonymous>)
    at compareVersions (/home/tiste/dev/github/jenkins-infra/evergreen/services/node_modules/compare-versions/index.j
    at tree.map (/home/tiste/dev/github/jenkins-infra/evergreen/services/cli/manifest-resolver.js:89:15)
    at Array.map (<anonymous>)
    at ManifestResolver.resolveTree (/home/tiste/dev/github/jenkins-infra/evergreen/services/cli/manifest-resolver.js
    at tree.map (/home/tiste/dev/github/jenkins-infra/evergreen/services/cli/manifest-resolver.js:118:18)
(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing insi id: 3)
(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections
info: Optional plugin which is not resolvable being dropped: perforce 1.3.9
```